nmigen_boards/icefeather.py

```diff
@@ -0,0 +1,69 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ice40 import *
+from .resources import *
+
+
+__all__ = ["ICEFeatherPlatform"]
+
+
+class ICEFeatherPlatform(LatticeICE40Platform):
+    device      = "iCE40UP5K"
+    package     = "SG48"
+    default_clk = "clk12"
+    resources   = [
+        Resource("clk12", 0, Pins("35", dir="i"),
+                 Clock(12e6), Attrs(GLOBAL=True, IO_STANDARD="SB_LVCMOS")),
+
+        RGBLEDResource(0,
+            r="47", g="41", b="39",
+            attrs=Attrs(IO_STANDARD="SB_LVCMOS")),
+
+        *LEDResources(pins="47", invert=True, attrs=Attrs(IO_STANDARD="SB_LVCMOS")),
+        # Semantic aliases
+        Resource("led_r", 0, PinsN("47", dir="o"), Attrs(IO_STANDARD="SB_LVCMOS")),
```

Comment:
What's going on with the LEDs here? If there's one RGB LED then there should be just an RGB LED resource.